mozjs-sys/build.rs

```diff
@@ -249,7 +220,7 @@ fn build_spidermonkey(build_dir: &Path) {
 
     #[cfg(feature = "intl")]
     {
-        let icu_c_include_path = get_icu_capi_include_path();
+        let icu_c_include_path = env::var("DEP_ICU_CAPI_C_INCLUDE_DIR").unwrap();
```

Comment:
This (Edit: the name after `DEP_`) needs to match the name from the `links` key, i.e. you need to add the `1` you added in the links key.